### DISCUSSION TITLE
Possible collaboration/merger with Photon?

### DISCUSSION THREAD

**Discussion by Limfjorden** [localsend/localsend#1099] [Discussion (English, International)]
I stumbled upon an app called Photon, that seems to do the exact same thing as LocalSend. It makes sense to me to merge these two projects (or at least collaborate), since they are so similar and are trying to achieve the same goal.

What do you all think of this?

[Photon](https://github.com/abhi16180/photon)

**Comment by Tienisto** [comment]
Hello, your topic is quite broad. In what area should be improved? In the open source space, different projects can coexist, especially because nothing is kept in secret.

A merge is unlikely because software projects cannot be merged easily. We are not companies and do not own any patents
